protos/in_toto_attestation/predicates/human_review/v0/human_review.proto

```diff
@@ -0,0 +1,16 @@
+syntax = "proto3";
+
+package in_toto_attestation.predicates.human_review.v0;
+
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/in-toto/attestation/go/predicates/human_review/v0";
+option java_package = "io.github.intoto.attestation.predicates.human_review.v0";
+
+message Review {
+    string result = 1;
```

Comment:
missing `Reviewer reviewer`?

Reply:
Nope, this generic one was intended to be signed by the reviewer themself, so it didn't include the reviewer field which is needed when github or gerrit issues the attestation.